src/Utils/hiopOptions.cpp

```diff
@@ -890,6 +890,44 @@ void hiopOptionsNLP::register_options()
                         "Selects the hardware architecture to run the Ginkgo linear solver on.");
   }
 
+ 
+  // choose triangular solver implementation in Ginkgo.
+  // - Default is 'sparselib' which uses vendor triangular solvers
+  // - 'syncfree' uses the busy waiting loop based Ginkgo implementation
+  {
+    vector<string> range {"syncfree", "sparselib"};
+
+    register_str_option("ginkgo_trisolve",
+                        "syncfree",
+                        range,
+                        "Selects the triangular solver for Ginkgo.");
+  }
+
+
+  // Ginkgo iterative refinement options
+  {
+    register_int_option("ir_inner_ginkgo_restart",
```

Comment:
Wouldn't make sense to consolidate the options for inner IR under the same name: `ir_inner_restart`, `ir_inner_tol`, and `ir_inner_maxit`. Currently we have `ir_inner_X_restart`, `ir_inner_X_tol`, and `ir_inner_X_maxit` for both X=GINKGO and X=cuSolver. But only one linear solver would be used/active at any given time (under the same NLP), isn't?

Reply:
Please double check that I did not miss any of the `ir_inner_cusolver_X` occurrences